client/python/eidolon_ai_client/events.py

```diff
@@ -122,6 +123,7 @@ class StringOutputEvent(OutputEvent):
 class ObjectOutputEvent(OutputEvent, Generic[T]):
     event_type: Literal["object"] = "object"
     content: T
+    hidden: Optional[bool] = False
```

Comment:
hidden becomes very ui specific. Perhaps it would be better to add a metadata object to any event, and then we can throw this in there?

If not, this shouldn't be Optional[bool], just make it a bool

Reply:
How about "internal"? There are, and will likely be more of, events we don't want any UI to render, like this one.

Reply:
if that is the case, shouldn't we just put it in a separate context? maybe even a context with the name eidolon-hidden which the ui treats differently? seems like something we can represent without adding a new concept.